UnityProject/Assets/Scripts/HealthV2/Living/CirculatorySystem/Toxins/ToxinBase.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
```

Comment:
Codacy found an issue: [Remove this unnecessary 'using'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6324510)